src/azure-cli/HISTORY.rst

```diff
@@ -13,6 +13,19 @@ Release History
 * Fixed issue where `az functionapp create` does not accept Standard_RAGRS storage account type.
 * Fixed an issue where az webapp up would fail if run using older versions of python
 
+**HDInsight**
+
+* BREAKING CHANGE: create: Renamed --storage-default-container to --storage-container and --storage-default-filesystem to --storage-filesystem
```

Comment:
@Juliehzl, please format it to be more readable when you conduct the releasing next week.